lib/events/modalOfferRequestEvents.js

```diff
@@ -3,13 +3,13 @@ if (Meteor.isClient) {
     Template.ModalOfferRequest.events({
         'click .offerRequestBtn': function(options) {
             var options = {
-                listing_title: $(".offerRequestListingName").text(),
-                img: $(".offerRequestImg").attr("src"),
-                price: $(".offerRequestPriceOriginal").text(),
-                offerprice: $(".listprice").val(),
+                listing_title: $(".listingItemTitle").text(),
```

Comment:
Identifier 'listing_title' is not in camel case.<br>Mixed double and single quotes.